File: Cpp/26_Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.empty()) return 0;
        
        int length = 1;
        for(int i=1; i<nums.size(); ++i){
            if(nums[length-1] != nums[i]){
                nums[length] = nums[i];
                ++length;
            }
        }
        
        return length;
    }
};

File: Kotlin/26_Remove_Duplicates_from_Sorted_Array.kt
class Solution {
    fun removeDuplicates(nums: IntArray): Int {
        if(nums.isEmpty()) return 0
        
        var length = 1
        
        for(i in 1 until nums.size){
            if(nums[length - 1] != nums[i]){
                nums[length] = nums[i]
                ++length
            }
        }
        
        return length
    }
}